src/test/scala/io/altoo/akka/serialization/kryo/CryptoCustomKeySerializationTest.scala

```diff
@@ -13,38 +15,53 @@ class KryoCryptoTestKey extends DefaultKeyProvider {
 }
 
 object CryptoCustomKeySerializationTest {
-  val config =
-    """
-    akka {
-      actor {
-        serializers {
-          kryo = "io.altoo.akka.serialization.kryo.KryoSerializer"
-        }
-        serialization-bindings {
-          "scala.collection.immutable.HashMap" = kryo
-          "[Lscala.collection.immutable.HashMap;" = kryo
-          "scala.collection.mutable.LongMap" = kryo
-          "[Lscala.collection.mutable.LongMap;" = kryo
-        }
-      }
-    }
-    akka-kryo-serialization {
-      post-serialization-transformations = aes
-      encryption {
-        aes {
-          key-provider = "io.altoo.akka.serialization.kryo.KryoCryptoTestKey"
-          mode = "AES/GCM/PKCS5Padding"
-          iv-length = 12
-        }
-      }
-    }
-    """
+  private val config = {
+    s"""
+       |akka {
+       |  actor {
+       |    serializers {
+       |      kryo = "io.altoo.akka.serialization.kryo.KryoSerializer"
+       |    }
+       |    serialization-bindings {
+       |      "scala.collection.immutable.HashMap" = kryo
+       |      "[Lscala.collection.immutable.HashMap;" = kryo
+       |      "scala.collection.mutable.LongMap" = kryo
+       |      "[Lscala.collection.mutable.LongMap;" = kryo
+       |      "${ScalaVersionRegistry.immutableHashMapImpl}" = kryo
+       |      "${ScalaVersionRegistry.immutableHashSetImpl}" = kryo
+       |    }
+       |  }
+       |}
+       |
+       |akka-kryo-serialization {
+       |  post-serialization-transformations = aes
+       |  encryption {
+       |    aes {
+       |      key-provider = "io.altoo.akka.serialization.kryo.KryoCryptoTestKey"
+       |      mode = "AES/GCM/PKCS5Padding"
+       |      iv-length = 12
+       |    }
+       |  }
+       |}
+       |""".stripMargin
```

Comment:
since indenting does not matter for typesafe config we do not need the | stripmargin